fastlane/lib/fastlane/actions/reset_simulator_contents.rb

```diff
@@ -2,6 +2,10 @@ module Fastlane
   module Actions
     class ResetSimulatorContentsAction < Action
       def self.run(params)
+        if Helper.xcode_at_least?("9")
+          UI.user_error!("resetting simulators currently doesn't work with Xcode 9, stay tuned as we are working to add support for all new tools.")
```

Comment:
Sorry for nitpicking, didn't see this before, could we also include `"\nYou can change the Xcode version to use using `sudo xcode-select -s /Applications/Xcode...app`"` as we did on the other message, to let people know how to change the selected Xcode version for command line apps? Most people are probably not aware.